storage/volumes.md

```diff
@@ -416,9 +408,9 @@ $ docker container rm nginxtest
 $ docker volume rm nginx-vol
 ```
 
-## Share data among machines
+## Share data with machines
```

Comment:
I think the intent here was to describe how to share data on multiple machines; not sure if "with" covers that.

Perhaps "between" could be an alternative?